cmd/suggestion/chocolate/v1beta1/Dockerfile

```diff
@@ -15,9 +15,7 @@ ENV PYTHONPATH ${TARGET_DIR}:${TARGET_DIR}/pkg/apis/manager/v1beta1/python:${TAR
 
 RUN apt-get -y update && \
     apt-get -y install git && \
-    if [ "${TARGETARCH}" = "ppc64le" ] || [ "${TARGETARCH}" = "arm64" ]; then \
     apt-get -y install gfortran libopenblas-dev liblapack-dev g++; \
-    fi && \
```

Comment:
To avoid the following error, we must install these packages for all platforms.

```sh
...
#0 73.28       gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c src/Halton.cpp -o build/temp.linux-x86_64-3.9/src/Halton.o
#0 73.28       error: command 'gcc' failed: No such file or directory
#0 73.28       [end of output]
#0 73.28   
#0 73.28   note: This error originates from a subprocess, and is likely not a problem with pip.
#0 73.29 error: legacy-install-failure
#0 73.29 
#0 73.29 × Encountered error while trying to install package.
#0 73.29 ╰─> ghalton
...
```